collector.go

```diff
@@ -123,6 +138,10 @@ func (cc *ChunkedCollector) Flush() error {
 		if err := cc.Collector.Collect(spanID, p...); err != nil {
 			errs = append(errs, err)
 		}
+		if cc.FlushTimeout != time.Duration(0) && time.Since(start) > cc.FlushTimeout {
```

Comment:
Style; why not just `cc.FlushTimeout != 0`? You just want to check it's a non-zero value before using it. I don't think the explicit type is helpful.